editor.planx.uk/src/pages/FlowEditor/components/Settings/ServiceSettings/FlowStatus/index.tsx

```diff
@@ -100,7 +103,23 @@ const FlowStatus = () => {
         </Typography>
       </SettingsSection>
       <SettingsSection background>
+        {isTrial &&
+          <Box sx={(theme) => ({
```

Comment:
Rather than a new `Box` style here, can our existing `WarningContainer` or `ErrorSummaryContainer` styled components can be repurposed editor-side here?

Reply:
Good shout - swapped out for the existing `WarningContainer` 👍